pkg/util/paramtable/component_param.go

```diff
@@ -862,6 +862,7 @@ type rootCoordConfig struct {
 	ImportTaskSubPath           ParamItem `refreshable:"true"`
 	EnableActiveStandby         ParamItem `refreshable:"false"`
 	MaxDatabaseNum              ParamItem `refreshable:"false"`
+	MaxGeneralCapacity          ParamItem `refreshable:"false"`
```

Comment:
Maybe we should make this parameter refreshable.

Reply:
changed